### DISCUSSION TITLE
Need help to check my Authentication status

### DISCUSSION THREAD

**Discussion by theGrav1ty** [community/community#114053] [GitHub Education] [answered]
### Hi everyone,

I get this information from (https://education.github.com/discount_requests/application)
![1](https://github.com/community/community/assets/74085283/8f2a1d2a-95f3-4bfc-bcc1-e302fbf2b618)
but, I got this letter from my school email-box
![2](https://github.com/community/community/assets/74085283/8b527a99-5d74-4601-ab3a-97c420b384ea)
I don't understand if it was successful or not. Can someone help me?

**Comment by samus-aran** [comment, marked as answer]
Hi @theGrav1ty  👋

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [You were last verified as a "null" on...](https://github.com/orgs/community/discussions/111352#null-last-verified-on-date)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
